src/subcommand/wallet/send.rs

```diff
@@ -1,18 +1,32 @@
-use {super::*, crate::outgoing::Outgoing, base64::Engine, bitcoin::psbt::Psbt};
+use {super::*, crate::outgoing::Outgoing, base64::Engine, bitcoin::{psbt::Psbt}};
+
+#[derive(Clone, Copy)]
+struct AddressParser;
+
+#[derive(Clone, Debug, PartialEq)]
+pub(crate) enum ParsedAddress {
+  Address(Address<NetworkUnchecked>),
+  ScriptBuf(ScriptBuf),
+}
+
+fn parse_address(arg: &str) -> Result<ParsedAddress, bitcoin::address::Error> {
+  Ok(ParsedAddress::Address(Address::from_str(arg)?))
+}
```

Comment:
```suggestion
```
Let's get rid of all of these. The burn command should just use the transaction builder directly with the correct `TxOut`, which can include an OP_RETURN script.